CHANGELOG.rst

```diff
@@ -2,6 +2,211 @@
+
+- APM: Introducing the Error Tracking Standalone config option. Only span chunks
+  that contain errors or exception OpenTelemetry span events are taken into
+  consideration by sampling.
+
+- Add new windows images for LTSC 2019 and LTSC 2022:
+  - `datadog-agent:7-servercore-ltsc2019-amd64`
+  - `datadog-agent:7-servercore-ltsc2022-amd64`
+  - `datadog-agent:7-servercore-ltsc2019-jmx-amd64`
+  - `datadog-agent:7-servercore-ltsc2022-jmx-amd64`
+  - `datadog-agent:latest-servercore-ltsc2019-jmx`
+  - `datadog-agent:latest-servercore-ltsc2022-jmx`
+  - `datadog-agent:latest-servercore-ltsc2019`
+  - `datadog-agent:latest-servercore-ltsc2022`
+  - `datadog-agent:7.X.Y-ltsc2019`
+  - `datadog-agent:7.X.Y-ltsc2022`
+  - `datadog-agent:7.X.Y-ltsc2019-jmx`
+  - `datadog-agent:7.X.Y-ltsc2022-jmx`
+  - `datadog-agent:7.X.Y-servercore-ltsc2019`
+  - `datadog-agent:7.X.Y-servercore-ltsc2022`
+  - `datadog-agent:7.X.Y-servercore-ltsc2019-jmx`
+  - `datadog-agent:7.X.Y-servercore-ltsc2022-jmx`
+  - `datadog-agent:latest-ltsc2019`
+  - `datadog-agent:latest-ltsc2022`
+
+- [ha-agent] Add haagent component used for HA Agent feature.
+
+- The cluster-agent now can collect pod disruption budgets from the cluster.
+
+- Added support for collecting container image metadata when running on a CRI-O runtime.
+
+- USM now monitors TLS traffic encrypted with Go TLS by default.
+  To disable this feature, set the `service_monitoring_config.tls.go.enabled`
+  configuration option to false.
+
+- USM now monitors traffic encrypted with Istio mTLS by default.
+  To disable this feature, set the `service_monitoring_config.tls.istio.enabled` configuration option to false. 
+
+- Introduced a new configuration variable `logs_config.http_protocol`, allowing users to enforce HTTP/1.1 for outgoing HTTP connections in the Datadog Agent. This provides better control over transport protocols and improves compatibility with systems that do not support HTTP/2.
+  By default, the log agent will now attempt to use HTTP/2 (unless a proxy is configured) and fall back to the best available protocol if HTTP/2 is not supported.
+
+- Added a new feature flag `enable_operation_and_resource_name_logic_v2` in DD_APM_FEATURES. Enabling this flag modifies the logic for computing operation and resource names from OTLP spans to produce shorter, more readable names and improve alignment with OpenTelemetry specifications.
+
+- Add support for PHP Single Step Instrumentation in Kubernetes (not enabled by default)
+
+
+.. _Release Notes_7.61.0_Enhancement Notes:
+
+Enhancement Notes
+-----------------
+
+- [ha-agent] Run HA enabled integrations only on leader Agent
+
+- [ha-agent] Add agent_group tag to datadog.agent.running metric
+
+- Cluster Agent: ``DatadogAgent`` custom resource, cluster Agent deployment, and node Agent daemonset manifests are now added to the flare archive when the Cluster Agent is deployed with the Datadog Operator (version 1.11.0+).
```

Comment:
Should this entry be moved to `CHANGELOG-DCA.rst` instead ?

Reply:
Good catch, fixed!